src/test/java/attendance/AbsenceRecordTest.java

```diff
@@ -0,0 +1,54 @@
+package attendance;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class AbsenceRecordTest {
+
+  @DisplayName("주말_결석_기록_생성_시_예외_발생")
+  @Test
+  void weekendAbsenceRecordTest() {
+    List<LocalDate> dates = List.of(
+        LocalDate.of(2025, 3, 1),
+        LocalDate.of(2025, 3, 2)
+    );
+    for (LocalDate date : dates) {
+      assertThatThrownBy(() ->
+          new AbsenceRecord(date)
+      ).isInstanceOf(IllegalArgumentException.class)
+          .hasMessage("주말 또는 공휴일에는 운영하지 않습니다.");
+    }
+  }
+
+  @DisplayName("공휴일_결석_기록_생성_시_예외_발생")
+  @Test
+  void holidayAbsenceRecordTest() {
+    assertThatThrownBy(() ->
+        new AbsenceRecord(LocalDate.of(2025, 12, 25))
+    ).isInstanceOf(IllegalArgumentException.class)
```

Comment:
```suggestion
    assertThatThrownBy(() -> new AbsenceRecord(LocalDate.of(2025, 12, 25)))
        .isInstanceOf(IllegalArgumentException.class)
```
람다식 이후로 line이 넘어가지 않는데 줄바꿈은 의미없어보여요.